src/components/Grid/SimpleGrid.stories.mdx

```diff
@@ -0,0 +1,346 @@
+import {
+  Meta,
+  Story,
+  Canvas,
+  ArgsTable,
+  Description,
+} from "@storybook/addon-docs/blocks";
+import { withDesign } from "storybook-addon-designs";
+
+import SimpleGrid from "./SimpleGrid";
+import Card, {
+  CardImage,
+  CardHeading,
+  CardContent,
+  CardActions,
+} from "../Card/Card";
+import {
+  CardImageRatios,
+  CardImageSizes,
+  CardLayouts,
+} from "../Card/CardTypes";
+import { HeadingDisplaySizes, HeadingLevels } from "../Heading/HeadingTypes";
+import Icon from "../Icons/Icon";
+import { IconNames, IconSizes } from "../Icons/IconTypes";
+import Image from "../Image/Image";
+
+import { getCategory } from "../../utils/componentCategories";
+import DSProvider from "../../theme/provider";
+
+<Meta
+  title={getCategory("SimpleGrid")}
+  component={SimpleGrid}
+  decorators={[withDesign]}
+  parameters={{
+    design: {
+      type: "figma",
+      url:
+        "https://www.figma.com/file/qShodlfNCJHb8n03IFyApM/Master?node-id=10975%3A16",
+    },
+    jest: ["SimpleGrid.test.tsx"],
+  }}
+  argTypes={{
+    className: { table: { disable: true } },
+    id: { table: { disable: true } },
+  }}
+/>
+
+# SimpleGrid
+
+| Component Version | DS Version |
+| ----------------- | ---------- |
+| Added             | `0.25.1`   |
+| Latest            | `0.25.1`   |
+
+<Description of={SimpleGrid} />
+
+The `SimpleGrid` component is used to render UI elements in a uniform grid layout, with the column widths and grid spacing fixed throughout the entire grid.
+
+<Canvas withToolbar>
+  <Story name="SimpleGrid Props">
+    {(args) => (
```

Comment:
Set a default for the "columns" prop. Makes it easier to use in the "controls" tab.

Reply:
The default value for the component is 3 (the NYPL style default) and the `columns` prop does not need to be passed.  However, if I set a default value for the actual control in the ArtgsTable, then that value shows up in the code example.  Is there a way around that?

Reply:
@EdwinGuzman ^

Reply:
I added a default, but I also added more infomration about the `columns` props to the component description.